Common/src/main/java/me/egg82/tfaplus/services/InternalAPI.java

```diff
@@ -802,22 +822,22 @@ private static boolean getLoginExpensive(UUID uuid, String ip, long ipTime, Jedi
         // SQL
         try {
             LoginData result = null;
-            if (sqlType == SQLType.MySQL) {
-                result = MySQL.getLoginData(uuid, ip, sql, storageConfigNode).get();
-            } else if (sqlType == SQLType.SQLite) {
-                result = SQLite.getLoginData(uuid, ip, sql, storageConfigNode).get();
+            if (ConfigUtil.getSQLType() == SQLType.MySQL) {
+                result = MySQL.getLoginData(uuid, ip, ConfigUtil.getSQL(), ConfigUtil.getStorageConfigNode()).get();
+            } else if (ConfigUtil.getSQLType() == SQLType.SQLite) {
+                result = SQLite.getLoginData(uuid, ip, ConfigUtil.getSQL(), ConfigUtil.getStorageConfigNode()).get();
             }
 
             if (result != null) {
-                if (debug) {
+                if (ConfigUtil.isDebugging()) {
                     logger.info(uuid + " " + ip + " found in storage. Value: " + result);
                 }
                 // Update messaging/Redis, force same-thread
-                Redis.update(result, ipTime, redisPool, redisConfigNode).get();
-                RabbitMQ.broadcast(result, ipTime, rabbitConnection).get();
+                Redis.update(result, ipTime, ConfigUtil.getRedisPool(), ConfigUtil.getRedisConfigNode()).get();
+                RabbitMQ.broadcast(result, ipTime, getRabbitConnection()).get();
```

Comment:
Not using the broadcast method, here?
Redis & Rabbit broadcast/update should probably be in one method, if we're going to make it a method anyway.

Reply:
I see the broadcast method here?